src/Components/Scenes/SideNavigation/RequestListItem/RequestListItem-Container.js

```diff
@@ -1,13 +1,14 @@
 import { connect } from 'react-redux';
 import RequestListItem from './RequestListItem';
 
-const mapStateToProps = () => {
+const mapStateToProps = ({ currentUser }) => {
   return {
     requestListItemsName: [
       { label: 'Inbox', link: '/requests' },
       { label: 'Sent', link: '/requests/sent' },
       { label: 'Archive', link: '/requests/archive' }
-    ]
+    ],
+    currentRole: currentUser.role.approver
```

Comment:
Just a thought: what about this use case?
```
role: {
  governance: true,
  approver: false
}
```
per Eric's comments in slack, the API will later be refactored such that if you're a governor you can't be an approver as well. The reason I bring this up is it'll have to be fixed later down the road to check both booleans

Reply:
Should the condition be `(currentUser.role.approver || currentUser.role.governor)` then?

Reply:
didn't event think of that, but yes that should cover every scenario

Reply:
Revised to check both cases good call 😎